pipeline/test_transform.py

```diff
@@ -0,0 +1,152 @@
+"""Tests for transform functions."""
+
+from transform import convert_unix_to_datetime, clean_data, transform_sales_data
+import pytest
+
+
+def test_convert_unix_to_datetime_valid():
+    """Tests that a valid Unix timestamp is converted to a string.."""
+
+    unix_timestamp = 1718801551.04217
+
+    assert convert_unix_to_datetime(unix_timestamp) == "2024-06-19 12:52:31"
+
+
+def test_convert_unix_to_datetime_invalid_type():
+    """Tests that a TypeError is raised if a float is not passed in."""
+
+    unix_timestamp = "171880151.04217"
+
+    with pytest.raises(TypeError) as exc_info:
+        convert_unix_to_datetime(unix_timestamp)
+        assert (
+            str(exc_info.value)
+            == "Expected a float for unix_timestamp, but got <class 'str'>"
+        )
```

Comment:
Could try mark parameterize instead of having individual tests

Reply:
Good idea! Will do now.